apps/site/src/app/layout.tsx

```diff
@@ -16,7 +16,7 @@ export default function RootLayout({
 }) {
 	return (
 		<html lang="en">
-			<body className={inter.className}>{children}</body>
+			<body className={fuzzy.className}>{children}</body>
```

Comment:
Suggestion [non-blocking]: if we're using Bootstrap for site styles, it would be better to override `$font-family-base` to reduce potentially confounding references (refer to HackAtUCI/HackUCI-Site#23)
We can look into this separately since this PR seems mainly focused on header and footer content.